### PR TITLE
Primitive functions for double + long

### PR DESCRIPTION
**Pull request by TobiasRoland**
Duplicated the ThrowingIntFunction + corresponding test and replaced int with double and long.

There was a failing test when executing the entire test suite - was this intentional? I've kept it as a separate commit.

### CODE REVIEW

File: src/test/java/com/pivovarit/function/ThrowingBiConsumerTest.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
     @Test
     void givenString_whenSortJava8_thenSorted() {
-        String sorted = "bdCa".chars()
+        String sorted = "bdca".chars()
```
Comment (TobiasRoland) [inline src/test/java/com/pivovarit/function/ThrowingBiConsumerTest.java]: I'm running java 8 (151), and when running the tests this failed. Was the uppercase C intentional?

Comment (pivovarit) [reply src/test/java/com/pivovarit/function/ThrowingBiConsumerTest.java]: Really weird, can you provide exact error that you got? All tests are run on each build, so this definitely should not sneak in

**Comment by pivovarit** [comment]
Looks good, just keep in mind that in the future I'm planning on replacing the manual labor of creating dedicated interfaces for primitives with a code generation utility.

One last thing, all of these should be public and not package-private

File: src/test/java/com/pivovarit/function/ThrowingBiConsumerTest.java (hunk 1 above)
Comment (TobiasRoland) [reply src/test/java/com/pivovarit/function/ThrowingBiConsumerTest.java]: I ran the test from IntelliJ, the outcome is:
```
org.opentest4j.AssertionFailedError: 
Expecting:
 <"Cabd">
to be equal to:
 <"abcd">
but was not.
Expected :abcd
Actual   :Cabd
```

- I don't see how the implementation would yield `abcd` as there's nothing down-casing the character in the stream

Comment (TobiasRoland) [reply src/test/java/com/pivovarit/function/ThrowingBiConsumerTest.java]: Ah - I think the problem is the CI job isn't running the tests, the output of (just picked one at random) Travis:

```
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Results :
Tests run: 0, Failures: 0, Errors: 0, Skipped: 0
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
```

https://travis-ci.org/pivovarit/throwing-function/jobs/507548530

**Comment by TobiasRoland** [comment]
@pivovarit Yep, no worries. It's not that much work to do it manually for now since it's just duplicating the existing functions and renaming a few things, so I'm happy to let it be overwritten by a superior tool once it's in place :)

**Pull request closed without merging**